spring-mvc/src/main/webapp/WEB-INF/jsp/application/list.jsp

```diff
@@ -4,7 +4,7 @@
     Author     : charlliz
 --%>
 
-<%@ page contentType="text/html;charset=UTF-8" pageEncoding="utf-8" trimDirectiveWhitespaces="false" session="false" %>
+<%@ page contentType="text/html;charset=UTF-8" pageEncoding="utf-8" trimDirectiveWhitespaces="false" session="true" %>
```

Comment:
You mean template? Well, we could have some pages that doesn't require session (like _about_ or _documentation_).